src/main/java/org/profit/telegrambot/TgBot.java

```diff
@@ -0,0 +1,208 @@
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.KeyboardRow;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+
+import java.util.*;
+
+import static org.profit.telegrambot.AntiHardcode.*;
+
+public class TgBot extends TelegramLongPollingBot {
+
+    @Override
+    public void onUpdateReceived(Update update) {
+
+
+        if (update.hasMessage() && update.getMessage().hasText()){
+            String chatId = update.getMessage().getChatId().toString();
+            String message = update.getMessage().getText();
+
+            if (message.matches("[1-9]")){
+                sendMessage(chatId, "Enter the page you want to see:", (ReplyKeyboardMarkup) null);
+            }
+
+            switch (message) {
+                case "Main Page", "/start", "Shop's Info" -> firstKeyboard(chatId, message);
+                case "Go Shopping", "Forward", "Back" -> keyboard(chatId, message);
+                case "My Cart" -> {
+                    for (Integer product : products) {
+                        sendPhoto(chatId, product);
+                    }
+                    sendMessage(chatId,"Go to payment?", (ReplyKeyboardMarkup) null);
+                }
+            }
+        } else if (update.hasCallbackQuery()) {
+
+            String chatId = update.getCallbackQuery().getMessage().getChatId().toString();
+            String callData = update.getCallbackQuery().getData();
+            int messageId = update.getCallbackQuery().getMessage().getMessageId();
+
+            if (callData.equals("Added")){
+
+                String answer = "Added " + addCounter + " items in total";
+                EditMessageText newMessage = new EditMessageText();
+                newMessage.setChatId(chatId);
+                newMessage.setMessageId(messageId);
+                newMessage.setText(answer);
+                addCounter ++;
+
+                addToCart();
+                try {
+                    execute(newMessage);
+                } catch (TelegramApiException e) {
+                    e.printStackTrace();
+                }
+            }
+        } else if (update.getMessage().hasPhoto()){
+
+            List<PhotoSize> photos = update.getMessage().getPhoto();
+            String fileId = Objects.requireNonNull(photos.stream()
+                    .sorted(Comparator.comparing(PhotoSize::getFileSize).reversed())
+                    .findFirst()
+                    .orElse(null)).getFileId();
+
+            System.out.println(fileId);
+        } // This is a method to get photo's fileId to use it later
+    }
+
+    /*public synchronized void sendMessage(String chatId, String text){
+        SendMessage message = new SendMessage();
+        message.setChatId(chatId);
+        message.setText(text);
+
+        try{
+            execute(message);
+        } catch (TelegramApiException e) {
+            e.printStackTrace();
+        }
+    }*/ // Currently in doubt whether this method is needed or not
```

Comment:
I would rather you talked to your teammate or even somebody more experienced than left such a commentary)

Reply:
And of course, found a compromise or a solution.